components/map/overview.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Overview
+page_title: Map Overview
+description: Discover the Blazor Map and explore the examples.
+slug: components/map/overview
+tags: telerik,blazor,map,overview
+published: True
+position: 0
+---
+
+# Map Overview
+
+The Map displays geospatial information organized in layers.
+
+The component provides tile layers, shape (vector) layers, bubble layers, and marker layers.
+
+## Creating Blazor Map
+
+1. Use the `TelerikMap` tag to add the component to your razor page.
+
+2. Add the `MapLayer` tag nested inside `MapLayers`.
+
+3. Set the `Type` property.
+
+4. Set the `Attribution` and `Subdomains` properties.
+
+5. Provide the `UrlTemplate` property.
+
+>caption A basic configuration of the Telerik Map.
+
+````CSHTML
+@* This code snippet showcases an example of a basic Map configuration. *@
+
+<TelerikMap>
+    <MapLayers>
+        <MapLayer Type="@MapLayersType.Tile"
+                  Attribution="@Attribution"
+                  Subdomains="@Subdomains"
+                  UrlTemplate="@UrlTemplate">
+        </MapLayer>
+    </MapLayers>
+</TelerikMap>
+
+@code {
+    public string[] Subdomains { get; set; } = new string[] { "a", "b", "c" };
+    public string UrlTemplate { get; set; } = "https://#= subdomain #.tile.openstreetmap.org/#= zoom #/#= x #/#= y #.png";
+    public string Attribution { get; set; } = "&copy; <a href='https://osm.org/copyright'>OpenStreetMap contributors</a>";
+}
+````
+
+## Layers
+
+Tha layers are responsible for organizing the Map information. [Read more about the supported Blazor Map layers...]({%slug components/map/layers%})
+
+## Markers
+
+You can add different points with real coordinates on the map as markers. [Read more about the Blazor Map Markers...]({%slug components/map/layers/marker%})
+
+## Zoom and Navigation
+
+Raster maps are divided into images (tiles) for serving over the web. Tiles are typically 256px squares. The top level (zoom level 0) displays the whole world as a single tile. Each progressive zoom level doubles the size of the Map. You can set the zoom level through the `Zoom` property. The zoom operation can be performed with a double click on the map or by using the mouse scroll wheel.
+
+Blazor Map incorporates a navigation tool allowing the end to user to easily zoom, pan and change the current view. You can change the navigation tool position by using the `MapControlsNavigator.Position` enum.
+
+## Pan
+
+The end user can pan the control by simply holding the left mouse button and dragging the map to a desired location.
```

Comment:
The sections can be combined as navigation is very similar to "panning". Something like "Pan and Zoom" sounds OK as people tend to search for them.